src/index.js

```diff
@@ -1,4 +1,5 @@
 export { default as Accordion } from './components/accordion';
+export { default as BarChart } from './components/charts/bar-chart';
```

Comment:
Can we use it directly or are we using the Chart component as a proxy and this is unnecessary?

Reply:
👍